src/ClientData/CallstackTypes.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ClientData/CallstackTypes.h"
+
+#include "OrbitBase/Logging.h"
+
+using orbit_client_protos::CallstackInfo;
+
+namespace orbit_client_data {
+
+std::string CallstackTypeToString(CallstackInfo::CallstackType callstack_type) {
+  switch (callstack_type) {
+    case CallstackInfo::kComplete:
+      return "Complete";
+    case CallstackInfo::kDwarfUnwindingError:
+      return "Dwarf unwinding error";
+    case CallstackInfo::kFramePointerUnwindingError:
+      return "Frame pointer unwinding error";
+    case CallstackInfo::kInUprobes:
+      return "Callstack inside uprobes (kernel)";
+    case CallstackInfo::kInUserSpaceInstrumentation:
+      return "Callstack inside user-space instrumentation";
+    case CallstackInfo::kCallstackPatchingFailed:
+      return "Callstack patching failed";
+    case CallstackInfo::kStackTopForDwarfUnwindingTooSmall:
+      return "Collected raw stack is too small";
+    case CallstackInfo::kStackTopDwarfUnwindingError:
+      return "Dwarf unwinding error in inner-frame";
+    case CallstackInfo::kFilteredByMajorityOutermostFrame:
+      return "Outermost frame error";
```

Comment:
We should come up with something slightly more specific for this case (regardless of the tooltip). This tells us that unwinding went wrong but the unwinder didn't realize, and we catch this by looking at the outermost frame. It's a bit of a "catch all" category. "Unknown error" or "generic error" might be better.